gcsio/src/main/java/com/google/cloud/hadoop/gcsio/GoogleCloudStorageImpl.java

```diff
@@ -469,7 +484,21 @@ public GoogleCloudStorageOptions getOptions() {
   @Override
   public WritableByteChannel create(StorageResourceId resourceId, CreateObjectOptions options)
       throws IOException {
+    Stopwatch stopwatch = Stopwatch.createStarted();
     logger.atFiner().log("create(%s)", resourceId);
+    try {
+      AbstractGoogleAsyncWriteChannel<?> channel = createInternal(resourceId, options);
+      recordSuccessMetric(metricsRecorder, LATENCY_MS, stopwatch, "create", PROTOCOL_JSON);
```

Comment:
probably `stopwatch` should be stopped ASAP in the parent methods to improve accuracy.

Reply:
Makes sense, it's cleaner that way as well